dbms/src/TestUtils/mockExecutor.cpp

```diff
@@ -96,6 +103,8 @@ DAGRequestBuilder & DAGRequestBuilder::mockTable(const String & db, const String
         TiDB::ColumnInfo ret;
         ret.tp = column.second;
         ret.name = column.first;
+        if (ret.tp == TiDB::TP::TypeNewDecimal)
+            ret.flen = 65;
```

Comment:
Just a hack, I will test more about decimal in following pr.

Reply:
add some comments?